service/README.md

```diff
@@ -74,24 +95,74 @@ command line tool.
 Opening a Session
 -----------------
 
-
-POSIX Signal Control
---------------------
-
-
-Session Save and Restore
-------------------------
-
-
-D-Bus APIs
-----------
-
-- io.github.geopm.TopoGetCache
-- io.github.geopm.PlatformGetGroupAccess
-- io.github.geopm.PlatformSetGroupAccess
-- io.github.geopm.PlatformGetUserAccess
-- io.github.geopm.PlatformGetAllAccess
-- io.github.geopm.PlatformGetSignalInfo
-- io.github.geopm.PlatformGetControlInfo
-- io.github.geopm.PlatformOpenSession
-- io.github.geopm.PlatformCloseSession
+A client process opens a session with the GEOPM service each time a
+PlatformIO object is created with libgeopm or libgeopmpolicy while the
+GEOPM systemd service is active.  This session is initially opened in
+read only mode.  If any calls are made to the D-Bus write APIs:
+
+    io.github.geopm.PlatformWriteControl
+    io.github.geopm.PlatformPushControl
+
+then the session is converted into a read/write mode session.  Only
+one read/write mode session is allowed at any time, and if a client
+attempts to begin a read/write session while another client has one
```

Comment:
Is there a way for a privileged third party (e.g., admin) to stop an existing session? Or is there another way to handle denial of service that exploits this rule? I guess the failsafe disconnector + single-tenant assumption might be sufficient, as long as we know it is guaranteed.

Reply:
e.g., I think we wouldn't be able to count on the destructor alone since that may not execute if the program does not terminate cleanly.